cmd/openshift-tests/e2e.go

```diff
@@ -340,6 +340,34 @@ var staticSuites = testSuites{
 		},
 		PreSuite: suiteWithProviderPreSuite,
 	},
+	{
+		TestSuite: ginkgo.TestSuite{
+			Name: "experimental/reliability/minimal/parallel",
```

Comment:
Why not combine these into a single run (serial + parallel will automaticalyl get run correctly)?

Reply:
i didn't realize it would do that automatically, sure i can combine.

Reply:
(but why do we have separate serial jobs today, then?)

Reply:
i mean i based this on:

```
	{
		TestSuite: ginkgo.TestSuite{
			Name: "openshift/conformance/parallel",
			Description: templates.LongDesc(`
		Only the portion of the openshift/conformance test suite that run in parallel.
		`),
			Matches: func(name string) bool {
				if isDisabled(name) {
					return false
				}
				return strings.Contains(name, "[Suite:openshift/conformance/parallel")
			},
			Parallelism:          30,
			MaximumAllowedFlakes: 15,
			SyntheticEventTests:  ginkgo.JUnitForEventsFunc(synthetictests.StableSystemEventInvariants),
		},
		PreSuite: suiteWithProviderPreSuite,
	},
	{
		TestSuite: ginkgo.TestSuite{
			Name: "openshift/conformance/serial",
			Description: templates.LongDesc(`
		Only the portion of the openshift/conformance test suite that run serially.
		`),
			Matches: func(name string) bool {
				if isDisabled(name) {
					return false
				}
				return strings.Contains(name, "[Suite:openshift/conformance/serial") || isStandardEarlyOrLateTest(name)
			},
			SyntheticEventTests: ginkgo.JUnitForEventsFunc(synthetictests.StableSystemEventInvariants),
		},
		PreSuite: suiteWithProviderPreSuite,
	},
```